src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/AbstractMemberScopedReferenceFinder.cs

```diff
@@ -49,7 +49,7 @@ protected sealed override bool CanFind(TSymbol symbol)
         return Task.CompletedTask;
     }
 
-    protected sealed override async ValueTask FindReferencesInDocumentAsync<TData>(
+    protected sealed override ValueTask FindReferencesInDocumentAsync<TData>(
```

Comment:
nearly all cases of FindReferencesInDocumentAsync became totally sync.  but i couldn't do 100%.  so the signature still returns a task.